REQUIRE

```diff
@@ -1,3 +0,0 @@
-julia 0.6
-ColorTypes
-FixedPointNumbers
```

Comment:
You need to keep REQUIRE around for attobot (see https://github.com/attobot/attobot/issues/50).